docs/docs/collaboration.md

```diff
@@ -0,0 +1,9 @@
+# Collaboration
+
+To collaborate with others, add them as collaborators in RStudio Connect. During
+publishing, they should provide their API key and will be able to choose a
+content location to publish to if the notebook title is the same.
+
+Additionally, you may share notebooks if appropriate.
+
+For additional information, please see the [Collaboration](https://docs.rstudio.com/connect/user/publishing/#publishing-collaboration) section of the User Guide.
```

Comment:
Since we're in the Collaboration section of this user guide, we might want to specifically reference this as the RStudio Connect User Guide.